ci-operator/config/openshift-knative/serving-operator/openshift-knative-serving-operator-release-0.11.yaml

```diff
@@ -0,0 +1,26 @@
+tag_specification:
+  name: '4.3'
+  namespace: ocp
+promotion:
+  namespace: openshift
+  name: knative-v0.11.1
+base_images:
+  base:
+    name: '4.3'
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+images:
+- from: base
+  to: knative-serving-operator
```

Comment:
No tests to run?

Reply:
Even though we run the tests as part of serverless-operator repo I think it's important to run the tests from this repo here as well.

Reply:
Alright. Discussed offline. It's not critical to run tests here because they're run as part of serverless-operator repo which embraces this serving-operator. We can add them later if required.